pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -58,16 +58,17 @@ type KubernetesMetaConfiguration struct {
 	ClusterLabels                          map[string]string     `json:"cluster_labels,omitempty"`
 	InheritedLabels                        []string              `json:"inherited_labels,omitempty"`
 	ClusterNameLabel                       string                `json:"cluster_name_label,omitempty"`
-	NodeReadinessLabel                     map[string]string     `json:"node_readiness_label,omitempty"`
+	NodeReadinessLabel        			   map[string]string	 `json:"node_readiness_label,omitempty"`
```

Comment:
What IDE or editor are you using? Please, use a Go and Python linter to format the code correctly :)

Reply:
Using Intellij, I thought I was tidying that up!